libsrc/apple2/detect80cols.s

```diff
@@ -0,0 +1,56 @@
+;
+; Colin Leroy-Mira, 27/05/2025
+;
+; Verify the presence of a 80 columns card in slot 3,
+; and publish a flag accordingly.
+;
+        .export         ext80_present
```

Comment:
I'm wondering what "ext80" stands for. It reminds me all the time of "extended 80 column card". But that's not what this here is about.

Reply:
Indeed it's not necessarily extended... I have a hard time coming up with a good variable name (it can't start with a number!) I don't know if c300_is_80col is better?

Reply:
I see. What about 'aux80col'? We're only interested in an 80 column board in the auxilary slot of Apple //e.